src/pytorch_lightning/CHANGELOG.md

```diff
@@ -5,12 +5,21 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 
+## [1.9.3] - 2023-02-21
+
+### Fixed
+
+- Fixed bug where `set_epoch` was not called for prediction dataloaders ([#16785](https://github.com/Lightning-AI/lightning/pull/16785))
```

Comment:
I noticed this fix was not complete. It will be with https://github.com/Lightning-AI/lightning/pull/16826

So we can remove this

```suggestion
```